public/app/features/dashboard/state/initDashboard.test.ts

```diff
@@ -264,6 +295,25 @@ describeInitScenario('Initializing existing dashboard', (ctx) => {
   it('Should initialize redux variables if newVariables is enabled', () => {
     expect(ctx.actions[2].payload.action.type).toBe(variablesInitTransaction.type);
   });
+
+  it('should log dashboard_loaded event', () => {
```

Comment:
event emitted once per query (here same panel but 2 queries A and B - different datasources too)